x-pack/metricbeat/module/googlecloud/stackdriver/metrics_requester.go

```diff
@@ -52,11 +52,13 @@ type stackdriverMetricsRequester struct {
 func (r *stackdriverMetricsRequester) Metric(ctx context.Context, m string) (out []*monitoringpb.TimeSeries) {
 	out = make([]*monitoringpb.TimeSeries, 0)
 
+	filter := r.getFilterForMetric(m)
+
 	req := &monitoringpb.ListTimeSeriesRequest{
 		Name:     "projects/" + r.config.ProjectID,
 		Interval: r.interval,
 		View:     monitoringpb.ListTimeSeriesRequest_FULL,
-		Filter:   r.getFilterForMetric(m),
+		Filter:   filter,
```

Comment:
nit: is this change necessary? 😬

Reply:
I might mess it up solving the conflicts. Glad that you spot it to take a closer look :grimacing:

Reply:
I think it was only that, I reverted it :grimacing: